config.default.json5

```diff
@@ -34,8 +32,8 @@
   autoPeeringPeriod: '*/10 * * * * *',
   diskUsageScanPeriod: '*/30 * * * * *',
   uploadLimitSizeBytes: 268435456, // 256 Mb.
-  findFileTimeout: 30000,
+  findFileTimeout: 20000,
   cors: {
-    originRegexps: ['.*\.adamant\.im', 'adm.im', '.*\.vercel\.app', '.*\.surge\.sh']
+    originRegexps: ['.*\.adamant\.im', 'adm.im', '.*\.vercel\.app', '.*\.surge\.sh', 'localhost:8080']
```

Comment:
I am not sure about localhost:8080 in production...

Reply:
For default config is ok. We can use other config for production, e.g. `config.prod.json5`.
https://github.com/Adamant-im/ipfs-node/blob/dev/src/config.ts#L9